store/config.go

```diff
@@ -59,36 +73,93 @@ func ParseConfig(filePath string, logger hclog.Logger) (*LogzioConfig, error) {
 	var logzioConfig *LogzioConfig
 	if filePath != "" {
 		logzioConfig = &LogzioConfig{}
+		// Set default values
+		logzioConfig.LogCountLimit = defaultLogCountLimit
+		logzioConfig.Compress = true
+		logzioConfig.InMemoryCapacity = defaultInMemoryCapacity
+		logzioConfig.InMemoryQueue = false
+		logzioConfig.DrainInterval = defaultDrainInterval
 		yamlFile, err := ioutil.ReadFile(filePath)
 		if err != nil {
 			return nil, err
 		}
 		err = yaml.Unmarshal(yamlFile, &logzioConfig)
+		if err != nil {
+			return nil, err
+		}
 	} else {
 		v := viper.New()
+		err := convertEnvironmentVariables()
+		if err != nil {
+			return nil, err
+		}
 		v.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
 		v.SetDefault(regionParam, "")
 		v.SetDefault(customAPIParam, "")
 		v.SetDefault(customListenerParam, "")
 		v.SetDefault(customQueueDirParam, "")
+		v.SetDefault(inMemoryQueueParam, false)
+		v.SetDefault(CompressParam, true)
```

Comment:
the same comment as previous regarding compress

Reply:
@Doron-Bargo, Yes. tracing is working fine with gzip compression, the traces are sent to logz.io and you can see them in the tracing UI.